src/coreclr/src/hosts/CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ if(CLR_CMAKE_HOST_WIN32)
   add_subdirectory(coreshim)
 else(CLR_CMAKE_HOST_WIN32)
   add_definitions(-D_FILE_OFFSET_BITS=64)
-  add_subdirectory(unixcoreruncommon)
+  include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Why did we need to touch corerun for this?

Reply:
The changes came from the original prototype. They are not strictly neededsince they were to support unixcorebundle. I removed most of that later.
This change and extracting of `GetCoreClrFunctions` method looked like refactoring that made things a bit cleaner, so I let it stay.

Reply:
It would be nice to submit unrelated refactoring's like this in separate PR.

Reply:
No problem. I can revert these. 
It was really mostly because I initially expected that much of the prototype code was going to stay.

Reply:
this was addressed - no longer changing anything in coreclr hosts